ITE/6/README.adoc

```diff
@@ -121,18 +134,23 @@ is equivalent to the following old-style link:
 [[motivation]]
 == Motivation
 
+This ITE was developed jointly by the in-toto maintainers and
+link:https://cloud.google.com/binary-authorization[Binary Authorization]. The
+framework aligns with the
+link:https://slsa.dev/attestation-model[SLSA Attestation Model]. The Provenance
+predicate type is the official SLSA recommendation.
```

Comment:
Suggested update to make this a bit more accurate. The old sentence about "the provenance predicate" was prior to that definition moving under SLSA.

```suggestion
This ITE was developed jointly by the in-toto maintainers, the
link:https://cloud.google.com/binary-authorization[Binary Authorization]
team, and what later became the
link:https://slsa.dev[SLSA]
team. The framework aligns with the
link:https://slsa.dev/attestation-model[SLSA Attestation Model]
and is officially recommended by SLSA.
```